packages/sdk-core/src/BacktraceCoreClient.ts

```diff
@@ -139,6 +139,12 @@ export abstract class BacktraceCoreClient {
      * @param report Backtrace Report
      */
     public async send(report: BacktraceReport): Promise<void>;
+
```

Comment:
the comment is invalid. We don't send Backtrace data anymore. Can we remove it and still use description from each send method above?

Reply:
Good catch, I removed the comment